test/cli/remote-table-upload/table-exist-columns-flag-diff-order/args

```diff
@@ -0,0 +1 @@
+remote table upload --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% --primary-key Id --file-name data.csv --file-tags test in.c-my-bucket.data data.csv --columns Status,Region,Id,Name,First_Order
```

Comment:
You still miss the test. Here is full set of columns, but we miss test where we have subset of columns in different order.

Reply:
[7dbf765](https://github.com/keboola/keboola-as-code/pull/1841/commits/7dbf765a29ed8b01a93b947b5744f7fdaa44536f)